runtime/runtime-params-estimator/src/cost.rs

```diff
@@ -60,15 +83,77 @@ pub enum Cost {
     ActionFunctionCallPerByte,
     ActionFunctionCallBaseV2,
     ActionFunctionCallPerByteV2,
+    /// Estimates `action_creation_config.transfer_cost` which is charged for
+    /// every `Action::Transfer`, the same value for sending and executing.
+    ///
+    /// Estimation: Measure a transaction with only a transfer and subtract the
+    /// base cost of creating a receipt.
     ActionTransfer,
+    /// Estimates `action_creation_config.stake_cost` which is charged for every
+    /// `Action::Stake`, a slightly higher value for sending than executing.
```

Comment:
It has been the case since at least protocol version 29 and I haven't been brave enough yet to dig into how parameter configuration worked before that.
Should we wait with this PR until I figure it out?